app/helpers/sticky_newsletter_visibility.rb

```diff
@@ -0,0 +1,15 @@
+module StickyNewsletterVisibility
+  def display_sticky_newsletter_on_this_slug?
+    @slug = request.url.gsub request.base_url, ''
```

Comment:
I would also prefer that we create a Component Class for this so we can keep all of the Sticky Newsletter UI Component's logic in one place.

There is GRASP Design Pattern called "Information Expert" which basically is about deciding where we put code/logic that responsible for certain software 'things'.

If we think about this, there is a strong case that the responsibility of deciding to show or hide the StickyNewsletter UI Component belongs inside a Class perhaps called StickyNewsletterUIComponent.

This would remove this code from the ApplicationController and keeps things nice and tidy.

The same should apply to the Inline Newsletter code and it's 'slug', currently this is also mixed up with the ApplicationController.